samples/IaC Mapping/azure-pipeline.yml

```
@@ -0,0 +1,27 @@
+trigger: none
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+- task: TerraformInstaller@0
+  inputs:
+    terraformVersion: '1.0.0'
+- checkout: self
+
+- task: AzureCLI@2
+  inputs:
+    azureSubscription: '<EnterServiceConnection>'
+    scriptType: 'bash'
+    scriptLocation: 'inlineScript'
+    inlineScript: |
+      az account show
+      cd ./Modules-Prod
+      terraform init
+      terraform plan 
+      terraform apply -auto-approve
+
+- task: MicrosoftSecurityDevOps@1
+  displayName: 'Microsoft Security DevOps'
+  inputs:
```

Since iacfilescanner runs by default, might be better to not put this so that all the other tools run too? Or putting this like so
```suggestion
  task: MicrosoftSecurityDevOps@1
  displayName: 'Microsoft Security DevOps'
  # If you want to only run iacfilescanner, uncomment the below lines
  # inputs:
      # tools: 'iacfilescanner'
```